cutscene_engine/cutscene_scenes/cutscene_test.tscn

```diff
@@ -0,0 +1,244 @@
+[gd_scene load_steps=15 format=2]
```

Comment:
Still a test file?

Reply:
Yes. cutscene_test.tscn was the main test scene to test two or more unique dialogues happening within a single level scene. probably keep it for good measures.

Reply:
We can add a DialogueStarter node when it is needed, it's not a problem. I think we can keep one test scene and delete the other, purely as an example, but please clean it up if possible.

Reply:
Could the need for multiple cutscene_tests be removed by using export on the dialogue_starter list and the json file path?

Reply:
> 
> 
> Could the need for multiple cutscene_tests be removed by using export on the dialogue_starter list and the json file path?

that could work actually! like an export variable?

Reply:
Yea just use something like export var dialogue_starter = []. There's an option for node paths in the array so you can use that. The json file could just be a string